x/evm/keeper/keeper.go

```diff
@@ -43,7 +43,8 @@ type Keeper struct {
 
 // NewKeeper generates new evm module keeper
 func NewKeeper(
-	cdc *codec.Codec, storeKey sdk.StoreKey, paramSpace params.Subspace, ak types.AccountKeeper,
+	cdc codec.BinaryMarshaler, storeKey sdk.StoreKey, paramSpace paramtypes.Subspace,
```

Comment:
can we use the codec.BinaryMarshaler that is set in `paramSpace` instead of `cdc`?

Reply:
that is a private codec